### ISSUE TITLE
[downgrade] reports path does not convert properly

### ISSUE THREAD

**Issue by jimsheldon** [drone/go-convert#137]
This v1 step
```yaml
    - name: store_test_results
      spec:
        reports:
        - path:
          - test-results.xml
          type: junit
        run: echo upload unit test results
      type: script
```
currently downgrades to this
```yaml
          - step:
              identifier: storetestresults
              name: store_test_results
              spec:
                command: echo upload unit test results
                reports:
                - spec:
                    paths:
                    - test-results.xml
                  type: junit
              timeout: ""
              type: Run
```
it should convert to this
```yaml
          - step:
              identifier: storetestresults
              name: store_test_results
              spec:
                shell: Sh
                command: echo upload unit test results
                reports:
                  type: JUnit
                  spec:
                    paths:
                      - test-results.xml
              timeout: ""
              type: Run
```

**Comment by jimsheldon** [comment]
I pushed a change to a test in this branch https://github.com/drone/go-convert/compare/issue-137

This test will pass when the required changes are made.

**Comment by jimsheldon** [comment]
Fix in #138
